sagemaker_processing/fairness_and_explainability/text_explainability_sagemaker_algorithm/byo_blazingtext_model_hosting.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<p>Add a Conclusion section summarizing what the customer accomplished in this notebook, and any suggested next steps.</p><p> </p><p>Add a Cleanup section deleting any resources that would continue to incur cost, in particular the <code>predictor</code> endpoint.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Done